### PR TITLE
Release administrivia: changelog stub, contributors, bump to v1.0

### PR DESCRIPTION
**Pull request by cortesi**

### CODE REVIEW

**Comment by Kriechi** [comment]
WebSocket support! Is that not worth mentioning?

**Comment by mhils** [comment]
Even bigger, mitmweb shall be official now! 🎉

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,24 @@
+18 December 2016: mitmproxy 1.0
+
+    * Mitmproxy is now Python 3 only
```
Comment (Kriechi) [inline CHANGELOG]: s/Mitmproxy/All mitmproxy tools/
This should include `mitmproxy`, `mitmdump`, `mitmweb`, `pathod`, and `pathoc`.

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+
+    * The config file format is now a single YAML file. In most cases,
+      converting to the new format should be trivial - please see the docs for
+      more information.
+
+    * console: significant UI improvements - including sorting of flows by
+      size, type and url, status bar improvements, much faster indentation for
+      HTTP views, and more.
+
+    * http2: significant improvements, but is temporarily disabled by default
```
Comment (Kriechi) [inline CHANGELOG]: Please capitalise: `HTTP/2`

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+      due to wide-spread protocol implementation errors on some large website
+
+    * websockets: the protocol implementation is now mature, and is enabled by
```
Comment (Kriechi) [inline CHANGELOG]: Please capitalise: `WebSocket`
Maybe mention that we have full support via hooks for messages and content manipulation.

**Pull request merged**